.gitignore

```diff
@@ -1,7 +1,58 @@
-# Created by https://www.toptal.com/developers/gitignore/api/intellij
-# Edit at https://www.toptal.com/developers/gitignore?templates=intellij
+# Created by https://www.toptal.com/developers/gitignore/api/intellij+all,node,vim,emacs,macos
```

Comment:
**Annotation**:
gitignore changes are primarily to take care of the new nodejs depenedency for docs builds.